src/sentry/api/endpoints/project_metrics.py

```diff
@@ -1,36 +1,70 @@
 from rest_framework.exceptions import ParseError
 from rest_framework.response import Response
 
-from sentry import features
 from sentry.api.bases.project import ProjectEndpoint
+from sentry.api.exceptions import ResourceDoesNotExist
 from sentry.snuba.metrics import DATA_SOURCE, InvalidField, InvalidParams, QueryDefinition
 
 
 class ProjectMetricsEndpoint(ProjectEndpoint):
-    """ Get metric name, type, unit and tag names """
+    """ Get metric name, available operations and the metric unit """
 
     def get(self, request, project):
-
-        if not features.has("organizations:metrics", project.organization, actor=request.user):
-            return Response(status=404)
-
         metrics = DATA_SOURCE.get_metrics(project)
         return Response(metrics, status=200)
 
 
+class ProjectMetricDetailsEndpoint(ProjectEndpoint):
+    """ Get metric name, available operations, metric unit and available tags """
+
+    def get(self, request, project, metric_name):
+        try:
```

Comment:
Do these endpoints need feature flag checks? Right now they look open to any organization.